pepper-apis/gae-egress-proxy/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+mkdir -p $SQUID_LOG_DIR
+chmod -R 755 $SQUID_LOG_DIR
+chown -R $SQUID_USER:$SQUID_USER $SQUID_LOG_DIR
+
+mkdir -p $SQUID_CACHE_DIR
+chown -R $SQUID_USER:$SQUID_USER $SQUID_CACHE_DIR
```

Comment:
Squid server kept running into permissions issues when these directories are initalized in the docker image, so I moved them here.

Reply:
What issues were you seeing?

Reply:
Don't have it on hand, but it was fine if I run the container by itself. But it's getting permissions errors when writing the logs files if I run it with a volume by `-v /var/log/squid:/var/log/squid` (e.g. mapping host log dir to container log dir). So something about that volume mount it's not happy with.